webui/react/package.json

```diff
@@ -29,14 +29,15 @@
     "io-ts": "^2.2.11",
     "js-sha512": "^0.8.0",
     "js-yaml": "^3.14.0",
-    "monaco-editor-webpack-plugin": "^2.0.0",
+    "monaco-editor": "^0.20.0",
```

Comment:
**question:** this wasn't there before (probably because `react-monaco-editor` depends on it and we don't use it directly). Do we need it now?

Reply:
I just removed this line and performed an `npm ci` and then gave it a shot and the editor works fine in the webui.

Reply:
react-monaco-editor does not specify a specific monaco-editor version and sets it to `*` and from the version matrix https://github.com/microsoft/monaco-editor-webpack-plugin#version-matrix I wanted to make sure we track the correct monaco-editor version. If this doesn't add to our bundle size or increase what npm downloads for dependencies keep it to so it's more clear what version we are getting?

Reply:
although package-lock would always show the version